app/src/main/cpp/native-lib.cpp

```diff
@@ -1,10 +1,190 @@
+
```

Comment:
There should be no "app" module.  All "aar" remnants should be removed from the library.  There is a reference to the ":app" module in the settings.gradle file that needs to be removed as well.